manifests/m/Meazure/GuardianBrowser/1.21.0.3/Meazure.GuardianBrowser.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Automatically updated by the winget bot at 2025/Feb/03
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Meazure.GuardianBrowser
+PackageVersion: 1.21.0.3
+Platform:
+- Windows.Desktop
+InstallerType: nullsoft
+UpgradeBehavior: install
+ProductCode: Guardian Browser 1.15.2

```

Comment:
This doesn't look right

Reply:
My guess would be Guardian Browser 1.21.0.3 but I'd have to install to be sure.

Reply:
I'm not entirely sure, but it appears to be missing:
![image](https://github.com/user-attachments/assets/779ec5ea-7472-4b68-a82f-e861003877c5)

Reply:
@Trenly what `ProductCode` do you think we should use here? Or should it be removed?

Reply:
@stephengillie - what registry key are those under?

Reply:
![image](https://github.com/user-attachments/assets/8e055d8c-0445-465d-9dd0-ac0d6aeb8d01)